### DISCUSSION TITLE
Module folder missing unable to edit a theme

### DISCUSSION THREAD

**Discussion by shivamchandra75** [JanDeDobbeleer/oh-my-posh#774] [Troubleshoot] [answered]
my posh theme is set to Paradox for my Powershell (windows Terminal) and it is working but there is no Module folder inside C:\Users\chand\Documents\WindowsPowerShell.
The paradox theme gets loaded correctly. please help

**Comment by JanDeDobbeleer** [comment]
How did you install oh-my-posh?

**Reply by shivamchandra75** [reply]
hello sir,through windows terminal in power shell 
I ran these commands
Install-Module posh-git                       
Install-Module oh-my-posh  

selected yes to all option in both

**Reply by JanDeDobbeleer** [reply, marked as answer]
You should be able to find the folder using `(Get-Module oh-my-posh).ModuleBase`

**Reply by shivamchandra75** [reply]
Thank you so much I found the folder.
please answer my question I am a complete beginner
1.)Is there a command to know the oh my posh version
2.) why module folder there while everybody else's is in documents/Powershell/Modules/

**Reply by JanDeDobbeleer** [reply]
1) `(Get-Module oh-my-posh).Version`
2) That's Powershell doing its thing, we have no influence on that. The question is why you need to know though 🤔

**Reply by shivamchandra75** [reply]
>1. thankyou for that sir 
>2. A very smart answer 😁.In case i face some issue in future related to this knowing that may help.
